### PR TITLE
Fixed urllib exeptions import after update requests to 2.6.0-8.el7_7

### PR DESCRIPTION
**Pull request by k0ste**

### CODE REVIEW

File: src/check_ceph_rgw_api [hunk 1]
```diff
@@ -62,7 +62,9 @@ def main():
       # Inversion of condition, when '--insecure' is defined we disable
       # requests warning about certificate hostname mismatch.
       if not args.insecure:
-          requests.packages.urllib3.disable_warnings()
+          from requests.packages.urllib3.exceptions import InsecureRequestWarning
```
Comment (valerytschopp) [inline src/check_ceph_rgw_api]: `from` statements must go at the top of the file

File: src/check_ceph_rgw_api [hunk 2]
```diff
@@ -62,7 +62,9 @@ def main():
...
+          from urllib3.exceptions import InsecureRequestWarning
```
Comment (valerytschopp) [inline src/check_ceph_rgw_api]: `InsecureRequestWarning` is imported from 2 different libraries ?!? Why ?

**Comment by k0ste** [comment]
@valerytschopp, I found a better way to filter warnings, but forgot to push change. Now it's done.

**Comment by valerytschopp** [comment]
@k0ste thanks for the fix

**Pull request merged**